LanguageFeatures/Unquoted-imports/path_separator_A01_t01.dart

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion Instead of `.`, this proposal uses `/`. This is obviously a
+/// natural path separator since it's the main path separator character used in
+/// most operating systems and in URLs on the web. Also, it is already the path
+/// separator character used inside Dart imports today.
+///
+/// @description Checks that it is a compile-time error to use `.` or `\` as a
+/// path separator.
+/// @author sgrekhov22@gmail.com
+
+// SharedOptions=--enable-experiment=unquoted-imports
+
+import test.package1.dotted.name1;
+//     ^^^^^^^^^^^^^^^^^^^^^^^^^^
+// [analyzer] unspecified
+// [cfe] unspecified
+
+import test.package1\dotted.name1;
+//     ^^^^^^^^^^^^^^^^^^^^^^^^^^
+// [analyzer] unspecified
+// [cfe] unspecified
+
+import test.package1/a\b/c;
+//     ^^^^^^^^^^^^^^^^^^^
+// [analyzer] unspecified
+// [cfe] unspecified
+
+import test.package1/a/b.c;
```

Comment:
So we're assuming that there is no `a/b.c.dart` in the package `test.package1`? It does create some amount of confusion that the test depends on so many other packages and libraries to exist or not exist. I think it would be helpful to have some comments to explain the situation.

Reply:
This test has been deleted